src/main/java/frc/robot/vision/VisionFilters.java

```diff
@@ -0,0 +1,55 @@
+package frc.robot.vision;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import frc.robot.constants.Field;
+import frc.robot.vision.data.AprilTagVisionData;
+import frc.robot.vision.data.VisionData;
+import frc.utils.Filter;
+import frc.utils.ToleranceCalculations;
+
+public class VisionFilters {
```

Comment:
Looks great, just one thing.
Make sure to test each one of the filters before the merge